src/message.rs

```diff
@@ -21,6 +21,8 @@ use std::{
 
 use crate::{client::Client, Headers};
 
+pub(crate) const MESSAGE_NOT_BOUND: &str = "message not bound to a client";
```

Comment:
client or connection?

Reply:
Probably connection since that is what they get when they connect. Client is more internal.

Reply:
technically it's Client, but I agree that for user it would make way more sense to call it `connection`